test/python/fx_importer/sparse_test.py

```diff
@@ -406,12 +417,13 @@ def forward(self, x):
 
     # Run it with PyTorch torch.sparse and with TORCH-MLIR sparse_jit.
     res1 = net(sparse_input)
+    res2 = sparse_jit(net, sparse_input)
     # TODO: make these work
-    # res2 = sparse_jit(net, sparse_input)
     # res3 = sparse_jit(net, batch_input)
     print("torch.sparse")
     print(res1)
     print("torch.mlir")
+    print(res2)
```

Comment:
Can you make this

   print(res2[0])
    print(res2[1])
    print(res2[2])

which prints the array a bit cleaner